_targets.R

```diff
@@ -153,4 +164,39 @@ list(
   # dh17 = dhfdc_q0.5
   # dh20 = dhfdc_q0.75
   
+  ########moving window nonstationarity stuff
+   ##table with all the FDC metrics computed on a moving window. The parameter min_yrs_in_window
+  ##screens out any moving windows for which there are too few years to be reliable. Can be an issue 
```

Comment:
You can make `min_yrs_in_window` a parameter. I don't think we'll change it from 15, but it could help to have the option. And these comments describing the function parameters should be moved to the function's R script

Reply:
Same comment for `min_windows`